tests/docs/multi_dut_asic_testing_support.md

```diff
@@ -0,0 +1,499 @@
+# Improve the test infrastructure to support multi-DUT and multi-ASIC systems
```

Comment:
this seems only addressing pytest infrastructure. I think somewhere we need to add a doc to describe the sonic testbed infrastructure support for that. for example, ansible deployment script, topology file.

How do we represent the multi-dut topology? minigraph generation?

Reply:
Yes, we'll need more documents to cover these areas. I will be working on them.